ToNote/Views/AboutPageView.xaml

```diff
@@ -6,7 +6,7 @@
              xmlns:local="clr-namespace:ToNote.Views"
```

Comment:
Both about page and add note dialog window are slightly broken. When you open either of them, there is a glitch in the gui (the style does not fully cover the new window, there is a black margin to the right and to the bottom)